ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -2895,6 +2895,156 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-vsphere-upi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
I see you've set it to optional: true, did you want this to run on every PR? If not should we start with manual triggering to save some resources?